k/kong/kong-3.3.0.patch

```diff
@@ -0,0 +1,412 @@
+ VENV = /dev/null # backward compatibility when no venv is built
+@@ -67,9 +67,7 @@ build-kong: check-bazel
+ build-venv: check-bazel
+ 	$(eval VENV := bazel-bin/build/$(BUILD_NAME)-venv.sh)
+ 
+-	@if [ ! -e bazel-bin/build/$(BUILD_NAME)-venv.sh ]; then \
+-		$(BAZEL) build //build:venv --verbose_failures --action_env=BUILD_NAME=$(BUILD_NAME); \
+-	fi
++	$(BAZEL) build //build:venv --verbose_failures --action_env=BUILD_NAME=$(BUILD_NAME)
+ 
+ install-dev-rocks: build-venv
+ 	@. $(VENV) ;\
+@@ -86,7 +84,6 @@ install-dev-rocks: build-venv
+ dev: build-venv install-dev-rocks bin/grpcurl
+ 
+ build-release: check-bazel
+-	$(BAZEL) build clean --expunge
+ 	$(BAZEL) build //build:kong --verbose_failures --config release
+ 
+ package/deb: check-bazel build-release
+diff --git a/build/nfpm/repositories.bzl b/build/nfpm/repositories.bzl
+index cc719072e..8939bc929 100644
+--- a/build/nfpm/repositories.bzl
++++ b/build/nfpm/repositories.bzl
+@@ -13,8 +13,8 @@ def _nfpm_release_select_impl(ctx):
+ 
+     if os_arch == "aarch64":
+         os_arch = "arm64"
+-    elif os_arch == "amd64":
+-        os_arch = "x86_64"
++    elif os_arch == "ppc64le":
++        os_arch = "ppc64le"
+     else:
+         fail("Unsupported arch %s" % os_arch)
+ 
+@@ -36,15 +36,16 @@ nfpm_release_select = repository_rule(
+ 
+ def nfpm_repositories():
+     npfm_matrix = [
+-        ["linux", "x86_64", "4c63031ddbef198e21c8561c438dde4c93c3457ffdc868d7d28fa670e0cc14e5"],
++        ["linux", "ppc64le", "4c63031ddbef198e21c8561c438dde4c93c3457ffdc868d7d28fa670e0cc14e5"],
+         ["linux", "arm64", "2af1717cc9d5dcad5a7e42301dabc538acf5d12ce9ee39956c66f30215311069"],
+-        ["Darwin", "x86_64", "fb3b8ab5595117f621c69cc51db71d481fbe733fa3c35500e1b64319dc8fd5b4"],
++        ["linux", "ppc64le", "4c63031ddbef198e21c8561c438dde4c93c3457ffdc868d7d28fa670e0cc14e5"],
++        ["Darwin", "ppc64le", "fb3b8ab5595117f621c69cc51db71d481fbe733fa3c35500e1b64319dc8fd5b4"],
+         ["Darwin", "arm64", "9ca3ac6e0c4139a9de214f78040d1d11dd221496471696cc8ab5d357850ccc54"],
+     ]
+     for name, arch, sha in npfm_matrix:
+         http_archive(
+             name = "nfpm_%s_%s" % (name, arch),
+-            url = "https://github.com/goreleaser/nfpm/releases/download/v2.23.0/nfpm_2.23.0_%s_%s.tar.gz" % (name, arch),
++            url = "https://github.com/goreleaser/nfpm/releases/download/v2.23.0/nfpm_2.23.0_%s_x86_64.tar.gz" % (name),
+             sha256 = sha,
+             build_file = "//build/nfpm:BUILD.bazel",
+         )
+diff --git a/build/nfpm/rules.bzl b/build/nfpm/rules.bzl
+index d6f5bb94f..84ce36ef3 100644
+--- a/build/nfpm/rules.bzl
++++ b/build/nfpm/rules.bzl
+@@ -9,8 +9,8 @@ def _nfpm_pkg_impl(ctx):
+     env = dicts.add(ctx.attr.env, ctx.attr.extra_env, KONG_VAR, ctx.configuration.default_shell_env)
+ 
+     target_cpu = ctx.attr._cc_toolchain[cc_common.CcToolchainInfo].cpu
+-    if target_cpu == "k8" or target_cpu == "x86_64" or target_cpu == "amd64":
```

Comment:
same thing here, why are we overwriting the arch? seems like this patch will break the x86 builds?